### ISSUE TITLE
--out_tensorname, --logit_tensorname, and --label_tensorname for COVIDNet-CXR Small and COVIDNet-CXR Large

### ISSUE THREAD

**Issue by Electro1111** [lindawangg/COVID-Net#114]
What are the correct arguments for these two models for training and inference purposes?

**Comment by Electro1111** [comment]
also --in_tensorname

**Comment by Electro1111** [comment]
I have found what I believe to be the correct tensors for these arguments, Im just not sure about the logit_tensorname

--out_tensorname dense_3/Softmax:0
--in_tensorname input_1:0
--logit_tensorname (dense_3/MatMul:0 or dense_3/BiasAdd:0)
--label_tensorname dense_3_target:0
--weights_tensorname dense_3_sample_weights:0

**Comment by haydengunraj** [comment]
Sorry for the late response, but I believe `dense_3/BiasAdd:0` is the correct logits tensor.
